# Goal: tham số đầu vào để di chuyển đến pose
string map_name
uint64 target_node_id
geometry_msgs/Pose2D current_pose
---
# Result: kết quả cuối cùng
bool success
string error_message
float64 total_distance
float64 total_time
---
# Feedback: trạng thái thời gian thực từ follow path
float64 distance_remaining
float32 speed